exercises/secret-handshake/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+Your task is to convert a number between 1 and 31 to a sequence of actions in the secret handshake.
+
+The sequence of actions are chosen by looking at the rightmost five digits of the number once it's been converted to binary.
```

Comment:
In the sentence as originally written, "sequence" is the subject, and "sequence" is singular, so verb "are" is inappropriate. Two ways to fix this:

```suggestion
The sequence of actions is chosen by looking at the rightmost five digits of the number once it's been converted to binary.
```

```suggestion
The actions are chosen by looking at the rightmost five digits of the number once it's been converted to binary.
```

Reply:
Thank you! I like both of the fixes, and I don't think it matters much, so I just picked one... If someone has a reason to prefer one over the other I'm happy to change my mind.